src/content/reference/manifest.mdx

```diff
@@ -717,6 +717,42 @@ volumes:
 
 Sets the working directory of your development container.
 
+### external (object, optional)
+
+A list of external resources to deploy with your development environment. This external resources contains the information for resources that are deployed outside the okteto cluster.
```

Comment:
Given that this iteration doesn't deploy anything, do we need to change the documentation?

```suggestion
A list of external resources that are part of your development environment.  Use this section for resources that are deployed outside of the okteto cluster, like Cloud resources or dashboards.
```